RPIPs/RPIP-36.md

```diff
@@ -0,0 +1,94 @@
+|   2023-06-10 | ShfRyn                           | Administrator, Non-Voting | [2023 Administrator Selection](https://dao.rocketpool.net/t/call-for-gmc-administrator-applications/1863/4)                          | [Address](https://etherscan.io/address/0x2bF5F18B21e78bAc8163ae8eF3B5Cd1530141570) |
+|   2023-03-10 | dondochaka                       | Voting                    | [2023 GMC Full Selection](https://vote.rocketpool.net/#/proposal/0xd0092d8d30e61ea3cdafdaeecbc81db8e84edbc045cf922365868de40f05ba13) | [Address](https://etherscan.io/address/0xeFe82d1eAf95191A28B8498688a4704D04efd52F) |
+|   2022-10-30 | jcrtp                            | Voting                    | [2023 GMC Full Selection](https://vote.rocketpool.net/#/proposal/0xd0092d8d30e61ea3cdafdaeecbc81db8e84edbc045cf922365868de40f05ba13) | [Address](https://etherscan.io/address/0xB1d7854b5C9c716C22F9659d93AE9ff7709b1bFC) |
+|   2022-10-30 | waqwaqattack                     | Voting                    | [2023 GMC Full Selection](https://vote.rocketpool.net/#/proposal/0xd0092d8d30e61ea3cdafdaeecbc81db8e84edbc045cf922365868de40f05ba13) | [Address](https://etherscan.io/address/0xd01b569900214B3864A588E09A711fbf3c506807) |
+|   2022-10-30 | Ken                              | Voting                    | [2023 GMC Full Selection](https://vote.rocketpool.net/#/proposal/0xd0092d8d30e61ea3cdafdaeecbc81db8e84edbc045cf922365868de40f05ba13) | [Address](https://etherscan.io/address/0x6212Ee7822265cE44B56C943bFd4bCcc03AeC42A) |
+
+### Historic Membership
+
+| Date Changed | Members                                                                                                   | Comment                                                                                                                                                                                               |
+|--------------|-----------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|   2023-11-10 | jcrtp, waqwaqattack, Ken, dondochaka, rplmaxi.eth, lookingforowls, FeelinggoodFeelingrt, epineph, rocknet | [2023 GMC Full Selection](https://vote.rocketpool.net/#/proposal/0xd0092d8d30e61ea3cdafdaeecbc81db8e84edbc045cf922365868de40f05ba13)                                                                  |
+|   2023-07-31 | jcrtp, waqwaqattack, Ken, mentor, objectObject, fornax, dondochaka, rplmaxi.eth, anisoptera.eth           | anisoptera.eth selected as [2022 GMC Expansion](https://vote.rocketpool.net/#/proposal/0x3431987750224c52ee4b1fd0510263ff05dbaee5b7258c078865042bffa03d8d) runner-up after Kevster steps down.        |
```

Comment:
Maybe "Promoted as next backup from [Full Selection] after Langers stepped down"? I don't like "selected" which makes it sound like someone picked.

a couple more instances in this table